src/main/java/hyperledger/besu/java/rest/client/listener/BlockEventListener.java

```diff
@@ -0,0 +1,26 @@
+package hyperledger.besu.java.rest.client.listener;
+
+import hyperledger.besu.java.rest.client.config.EthConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class BlockEventListener implements Runnable {
+
+    @Autowired private EthConfig ethConfig;
+
+    @Override
+    public void run() {
+        // TODO: Improve the logic to include all nodes from the network
+        // get events with all transactions data
+        ethConfig.getWeb3jList().get(0).blockFlowable(true)
```

Comment:
Couple of suggestions - 
1) How about listening to transaction and pending transactions events ?
2) How about giving the flexibity to the users to decide whether they want transactions inside blocks or not.
3) We can also leverage replay feature for replaying block and transaction history.

Reply:
1. Is it for reporting back the status of a transaction if it is still running/completed?
2. Sure, I will add a flag in upcoming PRs or after adding test cases.
3. Agree.